pkg/web_app/lib/src/widget/downloads_chart/widget.dart

```diff
@@ -47,6 +51,34 @@ void create(HTMLElement element, Map<String, String> options) {
     weeksToDisplay,
   );
 
+  final minorDisplayLists = prepareWeekLists(
+    data.totalWeeklyDownloads,
+    data.minorRangeWeeklyDownloads,
+    weeksToDisplay,
+  );
+
+  final patchDisplayLists = prepareWeekLists(
+    data.totalWeeklyDownloads,
+    data.patchRangeWeeklyDownloads,
+    weeksToDisplay,
+  );
+
+  final versionModesLists = {
+    'Major': majorDisplayLists,
+    'Minor': minorDisplayLists,
+    'Patch': patchDisplayLists
```

Comment:
I'd suggest making `value` and `label` two distinct things. Because one is display, the other is logic.
```suggestion
    'major': majorDisplayLists,
    'minor': minorDisplayLists,
    'match': patchDisplayLists
```

Reply:
done